recipe/meta.yaml

```diff
@@ -9,18 +9,18 @@ source:
   sha256: ebd79384c52569349edbeb85c0d7ed0dade8586c90cd8194c6c45d9f18017e37
 
 build:
-  number: 0
+  number: 1
   noarch: python
-  script: python -m pip install --no-deps --ignore-installed .
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-  build:
+  host:
     - python >=3
     - pip
   run:
     - python >=3
     - cartopy
-    - gdal 2.2.*
+    - gdal
```

Comment:
Just out of curiosity: is it actually needed to have this here? As gdal is not a direct dependency of geoplot, only indirect because of geoplot->geopandas->fiona->gdal

Reply:
Probably not now that we are using conda-build 3. I'll remove it in the next iteration.